dev/2018/spring/knights-to-kings/knights-to-kings.scss

```diff
@@ -0,0 +1,216 @@
+    background-color: $story-gold;
+    font-family: $serif;
+
+    h2 {
+      font-size: 28px;
+      margin-bottom: 20px;
+
+      @media(min-width: $screen-sm-min) {
+        font-size: 40px;
+      }
+    }
+  }
+
+  .heading {
+    color: $story-gold;
+    display: inline-block;
+    font-family: $sans-serif-cond;
+    font-size: 28px;
+    line-height: 1;
+    margin-bottom: 14px;
+  }
+
+  p {
+    font-size: 16px;
+    line-height: 1.4;
+
+    &.story-intro {
+      @media(min-width: $screen-sm-min) {
+        font-size: 18px;
+      }
+    }
+
+    &.byline {
+      font-style: italic;
+    }
+  }
+
+  .story-dropcap {
+    background-repeat: no-repeat;
+    background-size: 102px 143px;
+    margin-top: 10px;
+    padding-top: 10px;
+  }
+
+  .kings-court {
+    background-color: $story-black;
+    color: $story-white;
+    padding-bottom: 3rem;
+
+    .kings-court-header {
+      background-position: 50% 50%;
+      background-repeat: no-repeat;
+      color: $story-black;
+      display: block;
+      font-family: $serif;
+      font-size: 32px;
+      margin: -28px -20px;
+      padding: 10px;
+
+      @media(min-width: $screen-sm-min) {
+        left: calc(50% - 179px);
+        margin: 0;
+        padding: 30px;
+        position: absolute;
+        text-transform: uppercase;
+        top: -40px;
+        transform: rotate(-3deg);
+        width: 358px;
+      }
+    }
+
+    img {
+      @media(max-width: $screen-sm-min) {
+        max-width: 100px;
+      }
+    }
+
+    ul {
+      font-family: $sans-serif;
+      font-size: 12px;
+      font-weight: 400;
+      list-style: none;
+      margin-bottom: 2rem;
+      text-align: left;
+
+      @media(min-width: $screen-xs-min) {
+        font-size: 13px;
+        text-align: center;
+      }
+
+      li {
+        margin-bottom: 5px;
+      }
+    }
+  }
+
+  .splatter-left {
+    left: -290px;
+    position: absolute;
+    top: -140px;
+  }
+
+  .splatter-right {
+    position: absolute;
+    right: -290px;
+    top: -60px;
+    transform: scaleX(-1);
+  }
+
+  figure {
+    &.pull-left + p {
+      margin-bottom: 0;
+    }
+
+    figcaption {
+      font-family: $serif;
+      font-size: 30px;
+      line-height: 1;
+      text-align: center;
+      text-transform: uppercase;
+
+      .quoteby {
+        font-family: $sans-serif;
+        font-size: 15px;
+      }
+    }
+  }
+
+  .story-montage-image {
+    background-size: 140%;
+    min-height: 280px;
+
+    @media(min-width: $screen-xs-min) {
+      background-size: 100%;
+      min-height: 400px;
+    }
+
+    @media(min-width: $screen-md-min) {
+      min-height: 540px;
+    }
+
+    @media(min-width: 1200px) {
+      min-height: 680px;
+    }
+
+  }
+
+  .quoteby {
+    display: inline-block;
+    font-weight: 700;
+  }
+
+  .closing {
+    background-color: $story-black;
+    color: $story-white;
+    margin-top: -140px;
+    padding: 150px 0 50px;
+
+    @media(min-width: $screen-xs-min) {
+      margin-top: -240px;
```

Comment:
This negative margin-top needs to be fluid to match the scaling of the header image; the heading in `.closing` starts to clip into the header image between ~640-767px wide.

(However, if you're able to implement the alternative header image + linear gradient div I mentioned in Slack, you won't need these negative margins anymore anyway.)